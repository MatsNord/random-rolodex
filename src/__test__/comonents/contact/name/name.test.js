import React from 'react';
import renderer from 'react-test-renderer';
import Name from './../../../../components/contact/name/name';

test('NameItem render', () => {

  const tree = renderer.create(<Name />).toJSON();
  expect(tree).toMatchSnapshot();
});