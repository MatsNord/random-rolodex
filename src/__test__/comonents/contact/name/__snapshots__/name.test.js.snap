// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NameItem render 1`] = `
<span>
  <div
    className="name"
  />
</span>
`;
